### PR TITLE
WIP Make jira-cmd a cli tool

### PR DESCRIPTION
**Pull request by commuterjoy**
- [ ] Add some canned commands to create and list next bugs

### CODE REVIEW

**Comment by matthew-andrews** [comment]
does this need to be part of NBT?

File: package.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
   "version": "4.3.0",
   "bin": {
     "next-build-tools": "./bin/next-build-tools.js",
-    "nbt": "./bin/next-build-tools.js"
+    "nbt": "./bin/next-build-tools.js",
+    "jira": "./node_modules/jira-cmd/bin/jira.js"
```
Comment (matthew-andrews) [inline package.json]: npm does not guaranteed this path will always work……

Comment (commuterjoy) [inline package.json]: ok, good tip

**Comment by commuterjoy** [comment]
> does this need to be part of NBT?

Not especially :)

I could just document it on the new starter guide as an optional install.

**Comment by matthew-andrews** [comment]
> optional install
let's do that … NBT slows down builds already so much … trying to pare it down to the minimum ……

**Pull request closed without merging**